src/components/Timeinfo/Timeinfo.tsx

```diff
@@ -1,46 +1,31 @@
 import styles from './Timeinfo.module.scss';
-import { useEffect, useState } from 'react';
+import { useState } from 'react';
+import moment from 'moment';
+import 'moment-timezone';
 
 interface INewDate {
   time: Date;
+  tz: string;
+  difftime: string;
 }
-
 export default function Timeinfo(props: INewDate) {
-  const [time, setTime] = useState(props.time);
-
-  useEffect(() => {
-    const intervalId = setInterval(() => {
-      setTime(new Date());
-    }, 1000);
-
-    return () => {
-      clearInterval(intervalId);
-    };
-  }, []);
-
-  function formatTime() {
-    let hours = time.getHours();
-    const minutes = time.getMinutes();
-    const seconds = time.getSeconds();
-    const meridiem = hours >= 12 ? 'PM' : 'AM';
-
-    return `${padZero(hours)} : ${padZero(minutes)}: ${padZero(seconds)} ${meridiem}`;
-  }
-
-  function padZero(number:any){
-    return (number < 10 ? "0" : " ") + number;
-  }
+  const [time] = useState(props.time);
+  const [tz] = useState(props.tz);
+  const [difftime] = useState(props.difftime);
```

Comment:
```suggestion
  const [time] = useState(props.time);
  const [tz] = useState(props.tz);
  const [difftime] = useState(props.difftime);
```

Same comment about useState